symplyphysics/laws/thermodynamics/first_law_of_termodynamics.py

```diff
@@ -0,0 +1,35 @@
+from sympy import (Eq, solve)
```

Comment:
We have a separate folder for conservation related laws. What do you think, should it go to `conservation` instead of `thermodynamics`?